### PR TITLE
Suggested updates for v1.0.0

### PR DESCRIPTION
**Pull request by foogod**
Based on the discussion in #58, here's a PR to do the following:

- Fix the deprecated logging package reference in metrics.go
- Rename `ping_loss_percent` to `ping_loss_ratio`
- Change to use seconds by default (ms can still be selected by command-line switch)
- Update the version to 1.0.0

I've also gone ahead and:
- Turned off the deprecated metrics by default (can still be enabled if desired)
- Updated the README to reflect all the metric changes, and added a section on how to rename `ping_loss_ratio` back to `ping_loss_percent` in Prometheus if desired.
- Rearranged the README a bit to put relevant stuff under a "Changes from previous versions" section, so they're all easy to find if people are looking for that info (but they're out of the way if they're not).

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -38,10 +38,10 @@ var (
 
 var (
 	enableDeprecatedMetrics = true // default may change in future
-	deprecatedMetrics       = kingpin.Flag("metrics.deprecated", "Enable or disable deprecated metrics (`ping_rtt_ms{type=best|worst|mean|std_dev}`). Valid choices: [enable, disable]").Default("enable").String()
+	deprecatedMetrics       = kingpin.Flag("metrics.deprecated", "Enable or disable deprecated metrics (`ping_rtt_ms{type=best|worst|mean|std_dev}`). Valid choices: [enable, disable]").Default("disable").String()
 
 	rttMetricsScale = rttInMills // might change in future
-	rttMode         = kingpin.Flag("metrics.rttunit", "Export ping results as either millis (default), or seconds (best practice), or both (for migrations). Valid choices: [ms, s, both]").Default("ms").String()
+	rttMode         = kingpin.Flag("metrics.rttunit", "Export ping results as either seconds (default), or milliseconds (deprecated), or both (for migrations). Valid choices: [ms, s, both]").Default("s").String()
```
Comment (aqw) [inline main.go:44]: Very small nitpick: `Valid choices: [ms, s, both]")` should order the values as `[s, ms, both]` to match the order they are introduced in the text, and moves the default to the front (as is most common).

**Comment by aqw** [comment]
Thanks for putting this together @foogod!

We're evaluating this exporter to deploy at work, and from my view, this PR resolves all of our outstanding issues. +1 :-)

File: main.go (hunk 1 above)
Comment (foogod) [reply main.go]: Fixed!

**Review by czerwonk — APPROVED** [review]
LGTM

**Pull request merged**